ui-tests/src/test/java/io/syndesis/qe/steps/HomeSteps.java

```diff
@@ -0,0 +1,73 @@
+package io.syndesis.qe.steps;
+
+import static org.junit.Assert.assertThat;
```

Comment:
Seems like really a lot of unused imports

Reply:
fixed